src/framework/multiinstances/internal/ipc/ipc.h

```diff
@@ -29,10 +29,16 @@
 
 #include <functional>
 
+#include "muse_framework_config.h"
+
 class QLocalSocket;
 
 namespace muse::ipc {
-static const QString SERVER_NAME("musescore-app-ipc");
+#ifdef MUSE_APP_UNSTABLE
```

Comment:
might be better for there to be a separate processor symbol defined that is "development" (MUSE_APP_UNSTABLE) or ""?

Reply:
Perhaps... but then you might end up with `musescore-app-ipc-4-development` and `musescore-app-ipc-4-`, with trailing dash. So then you'd need to make the macro `-development`, but that makes it lose its generality, so that's far from ideal.

Anyway, we'll keep this in mind. The situation with "version labels", "build modes", "release channels" and what else, might deserve a small audit and potential cleanup, and maybe that will also lead to a change as you suggest. 

Fortunately, if we want/need to change this SERVER_NAME at any point, that's absolutely not the end of the world.